### PR TITLE
about: core team design revamp

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description
- Cleaned up the team section in `/about`
- Added missing `div`
- Pending removal of stale CSS code once I can get the mgmt commands working to populate the leaderboard
- Updated job opening section

screen shot 2018-04-18 at 5 31 49 pm

screen shot 2018-04-18 at 11 15 09 pm

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

- html
- css

### CODE REVIEW

File: app/retail/views.py [hunk 1]
```diff
@@ -70,15 +70,12 @@ def robotstxt(request):
 
 
 def about(request):
-    team_members_left = [
-        (static("v2/images/team/kevin-owocki.png"), "Kevin Owocki", "All teh things", "owocki", "The Community", "Avocado Toast"),
+    core_team = [
+        (static("v2/images/team/kevin-owocki.png"), "Kevin Owocki", "All the things", "owocki", "The Community", "Avocado Toast"),
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/retail/views.py]: Thanks @stickler-ci  :P

**Comment by thelostone-mc** [comment]
Pending: LinkedIn Profile of the core team so that I can link those up!

**Comment by owocki** [comment]
my linkedin profile => linkedin.com/in/owocki/

**Comment by owocki** [comment]
you can get others' on slack if u want

**Comment by thelostone-mc** [comment]
@owocki Yup got it ! @mbeacom was nice enough to gimme the links ^_^

Oh PS: Got the rotate + scale + change color effect you wanted 😂 
It looks neat

**Comment by thelostone-mc** [comment]
@mbeacom @PixelantDesign this look alright ?

**Comment by PixelantDesign** [comment]
Nice!

**Comment by darkdarkdragon** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.1 ETH (69.21 USD @ $692.1/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](http://localhost:8000/issue/gitcoinco/web/916/598).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $0 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Pull request merged**